src/app/app.module.ts

```diff
@@ -18,7 +18,7 @@ import { AutostartComponent } from './autostart/autostart.component';
     HomeComponent,
     MultiComponent,
     CustomComponent,
-    AutostartComponent
+    AutostartComponent,
```

Comment:
Isn't [that](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas) a [good practice](https://medium.com/@nikgraf/why-you-should-enforce-dangling-commas-for-multiline-statements-d034c98e36f8)?

Reply:
@Nek- Probably. Personally it isn't. It may be more convenient for some devs and it may clear the workflow up, but since it's not a something we're changing all the day I think it's not really useful. But to each his own...

Reply:
@maximelafarie the main goal of the trailing comma is when you add a new line you don't have to add the comma. So you just modify one line and don't pollute the history of the file

Reply:
@mcsky don't worry I understood. And I never said there's any bad side or not, you asked me my opinion, I give it. 🤓

Reply:
@maximelafarie btw json5 (pushed on the scene by typescript) allow trailingcomma https://json5.org/

Reply:
@mcsky @Nek- I'm an old school guy sometimes! 🤓Trailing commas in arrays are definitely useful. In objects, it could be discussed. 😋